besu/src/main/java/org/hyperledger/besu/cli/util/CommandLineUtils.java

```diff
@@ -99,10 +99,16 @@ public static void checkMultiOptionDependencies(
   private static String getAffectedOptions(
       final CommandLine commandLine, final List<String> dependentOptionsNames) {
     return commandLine.getCommandSpec().options().stream()
+        .filter(option -> Arrays.stream(option.names()).anyMatch(dependentOptionsNames::contains))
         .filter(
-            option ->
-                Arrays.stream(option.names()).anyMatch(dependentOptionsNames::contains)
-                    && !option.stringValues().isEmpty())
+            option -> {
+              try {
+                return !option.stringValues().isEmpty()
+                    || commandLine.getDefaultValueProvider().defaultValue(option) != null;
```

Comment:
Does this also work for environment variables?

Reply:
yep, default value provider in besu checks for environmental variables before config file.